src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/privateApi/jobs/ActivateLicencesController.kt

```diff
@@ -48,6 +48,6 @@ class ActivateLicencesController(
     ],
   )
   fun runLicenceActivationJob() {
-    return licenceActivationService.licenceActivationJob()
```

Comment:
made names a bit consistent

Reply:
Minor and non-blocking, should this be `activateLicences()` to match `deactivateLicences()`?

Reply:
it's a bit odd though as this does more than activate licences - it deactivates licences too! 

I called it licenceActivation to represent some more abstract process but didn't think about it too hard as its possibly doing too much